docs/managing-dragonfly/cluster-mode.md

```diff
@@ -25,6 +33,149 @@ True
+in this case you don't need to specify the `--cluster_mode` flag.
+
+In case you need to migrate data from Redis Cluster to Dragonfly you may find [Redis
+Riot](https://developer.redis.com/explore/riot/) helpful.
+
+## Multi Node Cluster
+
+Sometimes vertical scaling is not enough. In those cases, you'll need to set up a Dragonfly Cluster.
+
+A Dragonfly Cluster is similar to a Redis Cluster:
+
+* Multiple Dragonfly servers participate in a single logical data store
+* It provides all cluster-related commands required by Redis client libraries
+* It [distributes keys](https://redis.io/docs/reference/cluster-spec/#key-distribution-model) in the
+  same way Redis does
+* It supports [hash tags](https://redis.io/docs/reference/cluster-spec/#hash-tags)
+
+**Any client-side code that uses Redis Cluster should be able to migrate to Dragonfly Cluster with
+no changes.**
+
+However, setting up and managing a Dragonfly Cluster is different from managing a Redis Cluster.
+Unlike Redis, Dragonfly nodes do not communicate with each other (except for replication). Nodes are
+unaware of other nodes being unavailable, and cluster configuration is done separately to each node.
+
+Dragonfly only provides a _data plane_ (which is the Dragonfly server), but we do **not** provide a
+_control plane_ to manage cluster deployments.
+
+Follow the below steps to set up a Dragonfly Cluster
+
+### Start Dragonfly Nodes
+
+Start the Dragonfly nodes you need. Use the flags you would have used otherwise, but make sure to
+add the following (to both masters and replicas):
+* `--cluster_mode=yes` to let the nodes know that they are part of a cluster.
+* `--admin_port=X`, where `X` is some port that you'll use to run _admin commands_. This port should
+  generally not be exposed to users.
+
+Note: when started in cluster mode, nodes will reply with errors to any user requests until they are
+properly configured.
+
+### Configure Replication
+
+Replication is configured just like non-cluster replication, by using the `REPLICAOF` command. Note
+that this is a necessary step even though this information is present in the cluster configuration
+(see below).
+
+### Configure Nodes
+
+Note: Configuring cluster nodes is done by sending management commands via the _admin port_.
+Dragonfly will refuse to handle cluster management commands via the regular port.
+
+Cluster configuration includes information necessary for the cluster nodes to have in order to
+operate, like which nodes participate in the cluster (and in what role), which node owns which
+slots, etc.
+
+The first thing you'll need to do is to get each node's _id_, which is a unique string identifying
+the node. The id can be retrieved by issuing the command `DFLYCLUSTER MYID` on each of the nodes.
+
+Once you have all of the IDs, build the configuration string and pass it to all nodes using the
+`DFLYCLUSTER CONFIG <str>` command. It is important to pass the exact same configuration to all
```

Comment:
I think that you should add here that setting a config to a server will result in setting the slots to the server i.e it will reject keys not in its slot range and it will be able to response with this config to all cluster clients commands, but the server will not verify the other node or the replicas state, and also setting different slot configuration will not result in slot migration

Reply:
I added most of it here, but the migration was noted at the very bottom and so I emphasized it there